change/@fluentui-react-breadcrumb-027d665b-7783-4af2-86c0-5cbf7eaa5ec4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: moved sized icons to styling",
```

Comment:
also on second thoughs, we are adding a new API, shouldn't this be feat (minor) rather than patch ?

Reply:
We added only a state prop to the Divider but it won't affect API from a user side. What do you think, do we need to add it as minor in this case?

Reply:
proper semver is very tricky for UI.

from package API POV you're adding new API.
 - which is reflected in `read-breadcrumb.ap.md`, so it's a feature

from `Breadcrumb` component API POV nothing is changing besides internals, what is changing is the actual render output (UI) - which might be even considered as Breaking Change 🥹. I understand it is a "fix" based on Design Spec right ? so fix makes sense from that POV.


we should re-evaluate why we are exporting all the apis even from suite ( to improve that for v10)